src/tokens/Token.ts

```diff
@@ -230,9 +230,14 @@ class Token<P extends TokenPayload = TokenPayload> {
    * Exports this `Token` into `SignedToken`
    */
   public toSigned(): SignedToken<P> {
+    const clonedSignature = globalThis.structuredClone(this._signatures);
```

Comment:
I think we should put this into the `src/utils.ts` and then import it, that way it's clear what kind of `structuredClone` we are using.

Reply:
It was doing that in a utility function that was being used here.  But we're using the globally provided one only now since that's available. I recall you telling me to do this when we discussed it.

Reply:
Check https://github.com/MatrixAI/Polykey/pull/525#issuecomment-1631887853